packages/core-utils/src/time.ts

```diff
@@ -113,8 +121,15 @@ export function formatDuration(seconds) {
  * @param {object} region  an object that allows internationalization of the time
  * @returns {string}       formatted text representation
  */
-export function formatDurationWithSeconds(seconds, region) {
-  return formatDurationLikeMoment(seconds, { enabled: true, code: region });
+// TS TODO: region as type?
+export function formatDurationWithSeconds(
```

Comment:
We should deprecate this method.

Reply:
Deprecated in db1658fad93479daa48f7229371b9aef2200ae86